modules/example/resources/smithy4s.example.ObjectService.json

```diff
@@ -98,7 +98,11 @@
                         "name": "X-Foo",
                         "in": "header",
                         "schema": {
-                            "$ref": "#/components/schemas/LowHigh"
+                            "type": "string",
+                            "enum": [
+                                "Low",
+                                "High"
+                            ]
```

Comment:
Is this an intentional change to inline this enum? If so, why?

Reply:
No
I compiled example and received that

Reply:
or maybe my IDE did some wizardry, and i thought it was compiled output. Either  way should I remove?

Reply:
That's likely a result of some of your changes. Not sure which though, but I'd rather it wasn't inlined.